integration-tests/README.md

```diff
@@ -0,0 +1,81 @@
+# integration tests
+## Requirements
+The integration tests rely directly on docker-compose and assume
+that functionals `docker-compose`[^1] and `docker`[^2] commands are available in the environment.
+
+Required python dependencies are made available in the [test-requirements.txt](./test-requirements.txt) file.
+
+## Docker image
+The tests rely on a containerized deployment of the codebase.
+The container image can be built using the [Dockerfile](./docker/confgend.Dockerfile)
+available in the [docker](./docker) subdirectory.
+To build the image, this command can be run(assuming the current directory as the working directory):
+```
+docker build -f docker/confgend.Dockerfile -t wazoplatform/wazo-confgend-test:local ..
+```
+To test that the image is functional:
+```
+docker run --rm -it -v $(readlink ..):/usr/local/src/wazo-confgend wazoplatform/wazo-confgend-test:local
+```
+(note that the project root is mounted as a volume to provide in-development source code for testing).
+
+Once the image is made available locally, the docker-compose configurations should be able to use it and pull any other required images from the relevant registries.
+
+## Running tests
+
+The tests suite uses `pytest`[^3], provided through the [requirements file](./tests-requirements.txt).
+To run a single test file and all tests contained:
+```
+pytest suite/test_confgen.py
+```
+To run all tests:
+```
+pytest suite/
+```
+and to run only tests matching a specific marker and a specified name pattern:
+```
+pytest -m 'critical' -k 'pjsip_conf' suite/
+```
+
```

Comment:
Thought I understand the point, isn't the doc more likely to diverge if maintained separately from the object of the documentation? 
* I wrote this documentation from my intimate knowledge of this specific context. I cannot guarantee this applies to other repositories until I(or someone) goes through them all to validate. Isn't it easier to link down from those other repositories(i.e. delegate down instead of up)?
* occasional divergence is inevitable, keeping things consistent requires constant work. documenting divergences ensures they are made obvious and are properly observed and addressed(eventually if not immediately).
* while it is desirable to keep things consistent, isn't it more valuable to have something functional and appropriate in a specific context? It is possible to have things converge to consistency, through constant efforts. Guaranteeing a functional integration test setup with correct and sufficient documentation in a single repository should be easier and provides direct value to contributors.